<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>266</width>
    <height>317</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>点云滤波</string>
  </property>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>11</x>
     <y>75</y>
     <width>260</width>
     <height>191</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="page">
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>30</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>领域点数</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>70</y>
       <width>60</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>标准偏差</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stddev">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="meanK">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>121</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>30</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="page_2"/>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>241</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>去噪方法</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboBox">
        <item>
         <property name="text">
          <string>统计学</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>邻域点数滤波</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>comboBox</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>Form</receiver>
   <slot>filterMethodChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>181</x>
     <y>43</y>
    </hint>
    <hint type="destinationlabel">
     <x>599</x>
     <y>-61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>meanK</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Form</receiver>
   <slot>meanKChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>146</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>127</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>stddev</sender>
   <signal>textEdited(QString)</signal>
   <receiver>Form</receiver>
   <slot>stdDevChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>221</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>302</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>filterMethodChanged(int)</slot>
  <slot>meanKChanged(QString)</slot>
  <slot>stdDevChanged(QString)</slot>
 </slots>
</ui>
